cmd/nfd-worker/Dockerfile

```diff
@@ -1,4 +1,8 @@
 # Build node feature discovery
+#
+# NOTE: we keep the builder stage of nfd-master and nfd-worker Dockerfiles in
+# sync in order to re-use docker layers
+#
```

Comment:
now that we are here, change also the version of the base here

Reply:
Done